gamemodes/core/systems/police_system.inc

```diff
@@ -0,0 +1,210 @@
+#include <YSI_Coding\y_hooks>
+#include <YSI_Data\y_iterate>
+
+new
+    bool:PlayerCuffed[MAX_PLAYERS],
+    bool:PlayerJailed[MAX_PLAYERS]
+;
+
+static
+    Statement: stmt_retrieveJailed
+;
+
+static
```

Comment:
why is this in global scope, should be local only.

Reply:
Done.